### ISSUE TITLE
Does not work. 

### ISSUE THREAD

**Issue by corryalpha** [iklevente/WebhookDiscordTokenGrabber#3]
Discord closes, but no message is sent. I did it before, and it worked, it seems to be patched?

**Comment by iklevente** [comment]
Yes they've done something with the token management. It seems like it's still there but not on the same place.
I will be working on updating the code so it can find it on the new place.

**Comment by corryalpha** [comment]
Okay, I'm gonna close this issue once new commit is pushed.

**Comment by iklevente** [comment]
I was wrong. It works perfectly well. I thought that an update messed it up because I got random symbols and empty brackets the first time. Then I realised that I got two-factor authentication and that was the problem. When I turned it off it could grab my token. 

2-factor authentication is one of the biggest enemies of token grabbers and eventually it can be disabled, but it needs a lot of coding. I want to include this feature later in this project.

Maybe you have the same problem, if not I'm glad to help.

![tkngrbbr](https://user-images.githubusercontent.com/49818448/101995982-b73ba100-3cce-11eb-9355-4103adfd9374.PNG)
